addons/l10n_mx/data/account_configuration.yml

```diff
@@ -0,0 +1,10 @@
+-
+  Account settings set for Use Anglo-Saxon Accounting
+-
+  !record {model: account.config.settings, id: mx_default_account_properties}:
+    {
+      'use_anglo_saxon': 1,
```

Comment:
Well, I am not agreed with the last statement, again, As I mentioned before, the localization is not "multi-localization" the point is autoconfigure everything.

I can say the same with any module that overwrite any method.

If you are localizing a database what I expect is that all hidden features works as expected, anglo-saxon is one of them, (you can realize this was incorrectly missed Month after you started the operations and ugly things happen).